infrastructure/lambda/template.yaml

```diff
@@ -139,11 +139,11 @@ Mappings:
 
   IIQAPIStrategyMapping:
     Environment:
-      dev: 3 out of 4 prioritised
-      build: 3 out of 4 prioritised
-      staging: 3 out of 4 prioritised
-      integration: 3 out of 4 prioritised
-      production: 3 out of 4 prioritised
+      dev: "{\"2\": \"3 out of 4 prioritised\"}"
+      build: "{\"2\": \"3 out of 4 prioritised\"}"
+      staging: "{\"2\": \"3 out of 4 prioritised\"}"
+      integration: "{\"2\": \"3 out of 4 prioritised\"}"
+      production: "{\"2\": \"3 out of 4 prioritised\"}"
```

Comment:
Can we please use the [Fn::ToJsonString function](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-ToJsonString.html) from the [AWS::LanguageExtensions transform](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-languageextension-transform.html) to encode YAML objects as JSON strings and avoid escaping.

The language extenions transform must come before the `AWS::Serverless` transform in the list at the top of the template.

A usage example in one of our templates [here](https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/652ee49acb4d0dfa2c5a03403358ee914a424e4f/infrastructure/template.yaml#L869-L871)

Reply:
@CharlesIC will it work, with a map?

Reply:
Done, although I did have to put the language extension after the Serverless transform or it did not work ` Template format error: Unresolved resource dependencies [PublicKBVApiStage] in the Resources block of the template
`
I also refactored it as we discussed so it is no longer defined for each env.